### DISCUSSION TITLE
seperate system prompt for "Recursive-prompt. Use the prompt from last one🌀"

### DISCUSSION THREAD

**Discussion by tazztone** [xlinx/sd-webui-decadetw-auto-prompt-llm#25] [Ideas]
i think it would be good to have a 2nd sys prompt box for the "Recursive-prompt" feature.
i imagine using it like this:
normal sys prompt can be used for enhancing the "LLM your prompt" to get a more detailed prompt.
"recursive" can have a different purpose sys prompt like: "invert the meaning of the prompt" or "imagine the next scene of the story" etc.
![image](https://github.com/user-attachments/assets/82d67949-f77f-4f10-b344-1dc632c5227a)

what do you think of the idea?

oh and a selection of presets of different "system prompts" to choose from (and the ability to add your own) would be cool.

**Comment by xxlinxx** [comment]
okay... see u @tazztone again...
I just fix u mensioned Gemini issue and drop-down like function.
let me try rty . recursive v2
